src/events/guildCreate.js

```diff
@@ -8,7 +8,7 @@ class GuildCreate extends Event {
     async execute(guild) {
         if (!guild.available) return;
 
-        if (this.client.build === "stable") this.client.sendStatistics(message.guild.shardID);
+        if (this.client.build === "stable") this.client.sendStatistics(guild.shardID);
```

Comment:
I have pushed this directly to the master branch instead of the whole PR until it has been thoroughly tested. (9ad7e2d)